config.json

```diff
@@ -1,4 +1,4 @@
-﻿{
+{
```

Comment:
hmm... looks like the first line is still being changed. Not sure if that's related to the ide/text editor settings on your end.

Perhaps trying a different way to edit the file might work. If the issue persists I can send you the edited file on my end and see if that fixes the issue

Reply:
Actually my bad I was looking at the changes in the latest commit. The issue seems to have been fixed